course_catalog/task_helpers.py

```diff
@@ -431,3 +434,28 @@ def get_course_url(course_id, course_json, platform):
                 return preferred_urls[0].split("?")[0]
         return "{}{}/course/".format(settings.MITX_ALT_URL, course_id)
     return None
+
+
+def get_course_availability(course):
+    """
+    Gets the attribute `availability` for a course if any
+
+    Args:
+        course: Course model instance
+
+    Returns:
+        str: The url for the course if any
+    """
+    if course.platform == PlatformType.ocw.value:
```

Comment:
Can you add unit tests to cover this function? ocw vs edx, edx with and without course runs

Reply:
:+1: